website/docs/guides/sl-snowflake-qs.md

```diff
@@ -974,6 +974,81 @@ https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/snippets/_sl-co
 
 <ConnectQueryAPI/>
 
+## Connect and query with Hex
```

Comment:
hey @MichelleBaird-DBT , what do you think of having a single 'connect and query' section? and in that section, we have diff sub-sections for diff tools? Wondering if this approach allows us to easily add more tools in the future without creating more redundant top-level sections?

Reply:
![Screenshot 2024-07-18 at 15 17 28](https://github.com/user-attachments/assets/e2a53186-acc0-4c1e-9246-04c94c02d8e0)